sigar_freebsd.go

```diff
@@ -106,3 +107,7 @@ func parseCpuStat(self *Cpu, line string) error {
 	self.Idle, _ = strtoull(fields[4])
 	return nil
 }
+
+func (self *ProcTime) Get(pid int) error {
+	return ErrNotImplemented{runtime.GOOS}
```

Comment:
Please add an entry to the CHANGELOG and note the breaking changes to ProcTime on freebsd and openbsd.

Reply:
donz